### PR TITLE
add codes to justine.py

### PR DESCRIPTION
**Pull request by delima-justine**
please review my code, thank you!

### CODE REVIEW

File: justine.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        print('[2] Goals')
+        print('[3] Comments')
+        print('[4] Exit')
+        user_choice = int(input("Enter your choice: "))
+
+        match user_choice:
+            case 1: #  Basic Info
+                print('\nAge: 19')
+                print('Birthdate: February 24, 2005')
+                print('Currentt School Attended: Polytechnic of the Philippines-Taguig\n')
```
Comment (justine-skie) [inline justine.py]: Remove extra 't' to "Current"

**Review by justine-skie — CHANGES_REQUESTED** [review]
Please review the recommendations I made. Thank you!

File: justine.py (hunk 1 above)
Comment (delima-justine) [reply justine.py]: Fixed in commit 2a2726a90153b2b1104b360eaa45b4868e5556fb

**Review by justine-skie — APPROVED** [review]
Great!

**Pull request merged**